### PR TITLE
docs: add script for building standard lib docs

### LINKED ISSUE #383: Standard Library Cheatsheet
Hey guys, Merry Christmas! ☃

## Describe your issue/suggestions

Since we already have a [Syntax Cheatsheet](https://github.com/LingDong-/wenyan-lang#syntax-cheatsheet). I propose to adding a **Standard Library Cheatsheet**  helping users to better reference the functions in the standard library and contributors to quickly check if a function has been implemented already or not.

For example:

### [算經](https://github.com/LingDong-/wenyan-lang/blob/master/lib/%E7%AE%97%E7%B6%93.wy)
| Wenyan | Javascript Equivalent |
| --- | --- |
| `圓周率` | `Math.PI` |
| `正弦` | `Math.cos` |
| ... | ... |

## Before you open the issue, Please be sure ALL the items in the following list are checked

- [x] If this is a feature request, my request is NOT listed [Here](https://github.com/LingDong-/wenyan-lang#feature-requests)
- [x] If this is a bug report, my bug is not listed [Here](https://github.com/LingDong-/wenyan-lang#known-bugs)

### PR DESCRIPTION
**Pull request by antfu**
Resolves #383 

This approach adds a script to auto-generate standard lib docs from comments. The format would look like this:

```
疏曰「「餘弦。同犀之Math.cos也。」」
```

You can see the [**rendered Standard Library Cheatsheet here**](https://github.com/antfu/wenyan-lang/blob/docs/stdlib/documentation/Standard-Lib.md)

As I am poor in ancient Chinese, I can not tell what it is for many of the functions in the stdlib. (I think this will confuse many people as well).

I have only put some examples, and I would need the stdlib contributor to help to name the rest of them.

### CODE REVIEW

File: lib/算經.wy [hunk 1]
```diff
@@ -182,10 +183,12 @@
 	若非。乃得陽也。
 是謂「不可算數乎」之術也。
 
+疏曰「「下溢。同犀之Math.floor也。」」
```
Comment (statementreply) [inline lib/算經.wy]: This is a non-export function to generate floating underflow, not Math.floor

File: lib/算經.wy [hunk 2]
```diff
@@ -182,10 +183,12 @@
...
 吾有一術。名之曰「下溢」。欲行是術。必先得一數。曰「符」。乃行是術曰。
 	乘「符」以「微位冪」。乘其以「至微數」。乃得矣。
 是謂「下溢」之術也。
 
+疏曰「「下溢。同犀之Math.ceil也。」」
```
Comment (statementreply) [inline lib/算經.wy]: This is a non-export function to generate floating point overflow, not Math.ceil

File: lib/算經.wy [hunk 3]
```diff
@@ -182,10 +183,12 @@
 	若非。乃得陽也。
 是謂「不可算數乎」之術也。
 
+疏曰「「下溢。同犀之Math.floor也。」」
 吾有一術。名之曰「下溢」。欲行是術。必先得一數。曰「符」。乃行是術曰。
```
Comment (statementreply) [inline lib/算經.wy]: This is a non-export function (`吾有一術`) to generate floating point underflow, not Math.floor

File: lib/算經.wy [hunk 4]
```diff
@@ -182,10 +183,12 @@
...
 	乘「符」以「微位冪」。乘其以「至微數」。乃得矣。
 是謂「下溢」之術也。
 
+疏曰「「下溢。同犀之Math.ceil也。」」
 吾有一術。名之曰「上溢」。欲行是術。必先得一數。曰「符」。乃行是術曰。
```
Comment (statementreply) [inline lib/算經.wy]: This is a non-export function to generate floating point overflow, not Math.ceil

File: lib/算經.wy (hunk 3 above)
Comment (antfu) [reply lib/算經.wy]: @statementreply Sorry for that 😅. Removed. I am not very familiar with these. Can you help to add these comments and made a PR to `docs/stdlib` branch? Much appreciated!

**Comment by antfu** [comment]
Close, due to #432

File: lib/算經.wy (hunk 3 above)
Comment (statementreply) [reply lib/算經.wy]: The rounding functions are

算經 | JavaScript
:---: | :---:
取底 | Math.floor
取頂 | Math.ceil
取整 | similar to Math.round but different result for -(N+0.5)
捨餘 | Math.trunc

Comment (antfu) [reply lib/算經.wy]: Thanks! Would you like to edit the 算經.wy directly? Since there are many other functions not listed.

**Pull request closed without merging**